src/main/java/com/weaverplatform/importer/xmi/Main.java

```diff
@@ -16,8 +16,8 @@ public static void main(String[] args) throws IOException {
     
     if (args.length == 0){
       weaverUrl = "http://docker:9487";
-      filePath  = "/Users/mohamad/Desktop/InformatieBackboneModel.xml";
-      datasetId = "dataset0";  
+      filePath  = "/Users/bastiaan/Desktop/InformatieBackboneModel.xml";
+      datasetId = "model";  
     }
```

Comment:
These paths need to be either arguments to the program, or stored in a configuration file (neither travis-ci nor my laptop will have these paths).